lib/src/components/NxTreeView/NxTreeView.scss

```diff
@@ -18,16 +18,18 @@ $twisty-proportion-difference: 2px;
 */
 .nx-tree-view {
   box-sizing: border-box;
-  margin: var(--nx-spacing-4x) 0;
+  margin: var(--nx-spacing-4x) -8px;
```

Comment:
Should this use a spacing var?

Reply:
> Should this use a spacing var?

I didn't think that we could use a spacing var as a negative number.

Reply:
> > Should this use a spacing var?
> 
> I didn't think that we could use a spacing var as a negative number.

Feels hacky but uses the variable: https://github.com/sonatype/sonatype-react-shared-components/pull/408/commits/9b3afaf8deb50a3720d690413af3f00a0ddf7e3c